server/tests/ingest/parser/cp_transcripts.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+from unittest.mock import patch
+
+import flask
+import superdesk
+
+from cp.ingest import CPTranscriptsFeedParser
+
+from tests.ingest.parser import get_fixture_path
+from tests.mock import resources
+
+
+provider = {}
+parser = CPTranscriptsFeedParser()
+
+
+class CP_Transcripts_ParseTestCase(unittest.TestCase):
+    app = flask.Flask(__name__)
+
+    def test_parse(self):
+        with self.app.app_context(), patch.dict(superdesk.resources, resources):
+            superdesk.resources["archive"].service.find_one.side_effect = [
+                {"ingest_id": "d3c8487a-1757-4dde-8bb5-22ca166c1e67.1", "version": 2, "extra": {"ap_version": 999}},
+            ]
+            items = parser.parse(get_fixture_path("cp_transcripts.json", "cp_transcripts"), provider)
+            superdesk.resources["archive"].service.find_one.side_effect = None
+
+        item = items[0]
+        self.assertEqual("text", item["type"])
+        self.assertEqual("transcript", item["extra"]["type"])
```

Comment:
in theory this could be profile, but I guess superdesk wouldn't allow it if it doesn't exist right?

Reply:
Most likely, though I didn't want to create issues in the front end so I chose to use `extra` instead.
It's only set here and used in 1 function in newshub, so it's usage is quite minimal.